Akla/Extensions/UILabel+Extension.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  UILabel+Extension.swift
+//  Akla
+//
+//  Created by Ahmad Ayman Mansour on 18/05/2023.
+//
+
+import UIKit
+
+// MARK: - StyledLabel (UILabel Protocol)
+protocol StyledLabel {
+    func setTitle(_ title: String, textAlignment: NSTextAlignment?, fontSize: CGFloat?)
+    func setBody(_ body: String, textAlignment: NSTextAlignment?)
+}
+
+// MARK: - UILabel Extension for StyledLabel
+extension UILabel: StyledLabel {
+
+    func setTitle(_ title: String, textAlignment: NSTextAlignment?, fontSize: CGFloat?) {
```

Comment:
Please note that setting title || body style with apply all these configurations which are needed once. So, I belive you need to split styling logic from the text updating logic.

Reply:
is this better ? : 
`func setTitleStyle(textAlignment: NSTextAlignment? = .center , fontSize: CGFloat? = 30) { `